packages/fxa-auth-server/lib/senders/emails/templates/postVerifySecondary/index.txt

```diff
@@ -0,0 +1,7 @@
+postVerifySecondary-title = "Secondary email added"
+
+postVerifySecondary-description = "You have successfully added { $secondaryEmail } as a secondary email from your Firefox Account. Security notifications and sign-in confirmations will now be delivered to both email addresses."
+
+<%- include('/partials/manageAccount/index.txt') %>
+
+<%- include('/partials/changePassword/index.txt') %>
```

Comment:
I find this a bit strange but we're explicitly testing that the Mozilla support link is _not_ included in the plaintext version. I made a note in the "confirmation needed" doc to review later.